airflow/models.py

```diff
@@ -764,6 +770,8 @@ def command(
         cmd += "--local " if local else ""
         cmd += "--pool {pool} " if pool else ""
         cmd += "--raw " if raw else ""
+        cmd += "--airflow_cfg \"{pickled_cfg}\" " if airflow_cfg else ""
```

Comment:
@bolkedebruin @plypaul @aeon 
We were discussing the security of directly passing a pickled config through the command line. What are your guys' thoughts on this? 

There was also conversation about using multiprocessing to pass the config as an object and running the "raw" code path in `cli.py` directly so as to avoid running on the command line.